tests/test_integrated_channels/test_canvas/test_exporters/test_content_metadata.py

```diff
@@ -125,15 +125,15 @@ def test_transform_start(self, content_metadata_item, expected_start_value):
                 'full_description': 'Detailed description of edx demo course.',
             },
             '<a href=http://some/enrollment/url/>Go to edX course page</a><br />'
-            'Detailed description of edx demo course.'
+            'Detailed description of edx demo course. <br /><br />Starts: N/A<br />Ends: N/A'
```

Comment:
Not sure if this is totally necessary, but might be good to have a test that validates the dates themselves.